### ISSUE TITLE
Reskin: On client's page, Hovering mouse on Action button is not displaying the messages.

### ISSUE THREAD

**Issue by santoshmath** [openMF/community-app#2015]
Check the following screenshot from demo instance: where it shows meaning of action button(Approve in this case)

![reskin1](https://cloud.githubusercontent.com/assets/23022667/23607794/4f3664f8-028c-11e7-8810-602aeac8bd26.png)

But in reskin, meaning of action button is not displaying:

![reskin](https://cloud.githubusercontent.com/assets/23022667/23607837/803a1e8c-028c-11e7-81b0-441a25f48067.png)

**Comment by DanCarl857** [comment]
Wasn't able to reproduce this issue in the new skin.

**Comment by santoshmath** [comment]
I tested it on  new_reskin branch with latest code and newbeta. This issue still exists

**Comment by mbj36** [comment]
This was fixed today and it seems to be working fine for me.

On 6 Mar 2017 5:47 pm, "Santosh Math" <notifications@github.com> wrote:

I tested it on new_reskin branch with latest code and newbeta. This issue
still exists

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/openMF/community-app/issues/2015#issuecomment-284380995>,
or mute the thread
